### PR TITLE
feat(hub): cleanup angularjs catalog code

### PR DESCRIPTION
**Pull request by benjaminlepine**
ref: MANAGER-12587

| Question         | Answer
| ---------------- | ---
| Branch?          | feat/EPIC-MANAGER-12587
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12587
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
Can you also remove `packages/manager/modules/hub/src/components/catalog-items`?

**Review by darsene — APPROVED** [review]

**Comment by karim-ovh** [comment]
I think we should not remove the other language files. We should only delete the fr_FR files.

**Review by anooparveti — APPROVED** [review]

File: packages/manager/apps/hub/src/app.module.js [hunk 1]
```diff
@@ -106,6 +106,20 @@ export default async (containerEl, shellClient) => {
...
+          if (path === '/hub/catalog') {
+            return '/support';
+          }
+          if (/^\/ticket\//.test(path)) {
+            return path.replace(/^\/ticket\//, '/support/tickets/');
+          }
+          return undefined;
+        });
+      },
+    )
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/app.module.js:122]:
```suggestion
    .run(
      /* @ngInject */ ($urlRouterProvider) => {
        $urlRouterProvider.rule(($injector, $location) => {
          const path = $location.path();
          if (path === '/hub/catalog') {
            return '/catalog';
          }
          return undefined;
        });
      },
    )
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Comment by anooparveti** [comment]
Rebasing with `release/catalog-uapp`

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Comment by anooparveti** [comment]
Rebasing with `release/catalog-uapp`

**Pull request merged**